roles/ceph-defaults/defaults/main.yml

```
@@ -546,34 +546,31 @@ os_tuning_params:
 ceph_tcmalloc_max_total_thread_cache: 0
 
 
-##########
-# DOCKER #
-##########
+#############
+# CONTAINER #
+#############
 container_exec_cmd:
-docker: false
-ceph_docker_image: "ceph/daemon"
-ceph_docker_image_tag: latest
-ceph_docker_registry: docker.io
-## Client only docker image - defaults to {{ ceph_docker_image }}
-ceph_client_docker_image: "{{ ceph_docker_image }}"
-ceph_client_docker_image_tag: "{{ ceph_docker_image_tag }}"
-ceph_client_docker_registry: "{{ ceph_docker_registry }}"
-ceph_docker_enable_centos_extra_repo: false
-ceph_docker_on_openstack: false
+ceph_container_image: "ceph/daemon"
```

I wonder whether we would have to keep some backward compatibility like:

```suggestion
ceph_container_image: "{{ ceph_docker_image if ceph_docker_image is defined else 'ceph/daemon' }}"
```
or perhaps:

```suggestion
ceph_container_image: "{{ ceph_docker_image | default('ceph/daemon') }}"
```

---

I think that's a good idea to keep backward compatibility but do you know the variable list to apply this change ?
Because I don't know if we need to do this on all variables or only some of them